pkg/kubelet/dockershim/naming.go

```diff
@@ -66,6 +66,7 @@ func makeSandboxName(s *runtimeapi.PodSandboxConfig) string {
 	}, nameDelimiter)
 }
 
+// makeContainerName generates a container name that's guaranteed to be unique on its host.
```

Comment:
I just came across `randomizeName` below. So I guess the above statement in the comment is not guaranteed. Sorry for contradictory comments earlier where I thought  `makeContainerName` will always generate unique names.

Reply:
Joke is on me for not double-checking :)